modules/data-loading/pages/kafka-loader/kafka-ssl-sasl.adoc

```diff
@@ -0,0 +1,218 @@
+= SSL/SASL Authentication for Kafka Loader
+:description:
+
+TigerGraph's Kafka loader supports authentication and encryption using SSL and SASL protocols.
+This page details how to set up the following authentication and encryption protocol between the server (the external Kafka cluster) and the client (your TigerGraph instance).
+
+* Set up one-way SSL
+* Set up two-way SSL
+* Set up SASL
+* Set up SSL and SASL
+
+== Set up one-way SSL
+You can set up SSL to only have the client (TigerGraph) verify the identity of the server (your external Kafka cluster or broker).
+
+=== Before you begin
+The following prerequisites should be met on the server side:
+
+* Ensure you have generated a Certificate Authority (CA) on the server.
+* Ensure you have generated a certificate and private key with the server's hostname as the alias on the server.
+The certificate has been signed by the CA.
+* Ensure the IP address and the fully qualified domain name (FQDN) of the server is listed in the `/etc/hosts` file on the server.
+
+You will need the following files in order to configure SSL on TigerGraph.
+
+* The truststore file from the broker
+* The broker's Certificate Authority
+
+=== Procedure
+. Copy the following files from the server side to your TigerGraph cluster's m1 machine:
```

Comment:
We should copy files from server Kafka broker to all TigerGraph nodes instead of m1 machine.